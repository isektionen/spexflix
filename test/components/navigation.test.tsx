import React from 'react'
import { render } from '../testUtils'
import Navigation from '../../components/navigation'

describe('Footer component', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<Navigation />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
