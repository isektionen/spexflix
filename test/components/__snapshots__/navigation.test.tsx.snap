// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<DocumentFragment>
  <nav
    class="navigation"
  >
    <ul
      class="navList"
    >
      <li
        class="navItem"
      >
        <a
          class="navLinkActive"
          href="/"
        >
          Home
        </a>
      </li>
    </ul>
  </nav>
</DocumentFragment>
`;
